packages/core/src/auth/tokens/token-scope.ts

```diff
@@ -35,8 +77,11 @@ export class Permissions {
 }
 
 export const AllDataReadWrite: Permissions = new Permissions([
-  new CachePermission(CacheRole.ReadWrite),
-  new TopicPermission(TopicRole.ReadWrite),
+  new CachePermission(CacheRole.ReadWrite, {cache: new All()}),
```

Comment:
not that I mind this being explicit but technically you shouldn't need to do this, right? Isn't it the default?

Reply:
yes, it is the default so not needed technically.

Reply:
Removed the explicit arguments for All and also the redundant unit test I had added, since the unit test for `AllDataReadWrite` will also test we are using the right defaults.